### DISCUSSION TITLE
How can I calculated centre of mass rdf of aromatic ring from any peptide which containing aromatic ring using code and selecting atom of that ring?

### DISCUSSION THREAD

**Discussion by azizsimx** [MDAnalysis/mdanalysis#4701] [Q&A] [answered]
For example, a peptide containing tryptophan which has an indole ring, Now how can I calculate the center of mass of that ring and then it's position to calculate the ring-to-ring center of mass rdf plot?

**Comment by orbeckst** [comment, marked as answer]
First you need to learn how to select the atoms that you're interested in — see [Selections](https://userguide.mdanalysis.org/stable/selections.html) in the User Guide. This selection will give you an AtomGroup. 

Then you can compute the center of mass of your AtomGroup with the [AtomGroup.center_of_mass()](https://docs.mdanalysis.org/stable/documentation_pages/core/groups.html#MDAnalysis.core.groups.AtomGroup.center_of_mass) method for each frame of the trajectory. 

Once you have the coordinates of all centers that you want to calculate a RDF for, you need to write your own code because I am not aware of code in MDA that does it for you. You can look at the [RDF code for atoms](https://docs.mdanalysis.org/stable/documentation_pages/analysis/rdf.html). In particular you need to first calculate all distances with code similar to https://github.com/MDAnalysis/mdanalysis/blob/7618e05176b5f2c3008f375f840396ea8592488a/package/MDAnalysis/analysis/rdf.py#L269-L272 and then histogram the distances https://github.com/MDAnalysis/mdanalysis/blob/7618e05176b5f2c3008f375f840396ea8592488a/package/MDAnalysis/analysis/rdf.py#L287-L288

This is going to be some work but should be do-able with the existing examples.

**Reply by azizsimx** [reply]
Thank you for your response and sorry for the late reply I am able to solve that issue. Thank you again!!
